gecoscc-installer.sh

```diff
@@ -177,38 +178,44 @@ NO)
   esac
 fi
 
-
-echo "Adding EPEL repository"
+#echo "Adding EPEL repository"
 if ! rpm -q epel-release-6-8.noarch; then
-    rpm -ivh http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
+#if ! rpm -q epel-release-7-9.noarch; then
+    rpm -ivh --nosignature http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
+    #rpm -ivh --nosignature https://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-9.noarch.rpm
 fi
-echo "Installing python-devel and pip"
-install_package python-devel 
-install_package python-pip
+echo "Installing python2.7"
+yum install -y centos-release-SCL
+yum install -y python27
+source /opt/rh/python27/enable
 echo "Creating a Python Virtual Environment in /opt/gecosccui-$GECOSCC_VERSION"
 pip install virtualenv
 cd /opt/
-virtualenv gecosccui-$GECOSCC_VERSION
+virtualenv -p /opt/rh/python27/root/usr/bin/python2.7 gecosccui-$GECOSCC_VERSION
 echo "Activating Python Virtual Environment"
 cd /opt/gecosccui-$GECOSCC_VERSION
 export PS1="GECOS>" 
-source bin/activate
+source /opt/gecosccui-$GECOSCC_VERSION/bin/activate
+echo "Updating pip"
+pip install --upgrade pip
 echo "Installing gevent"
-pip install "https://pypi.python.org/packages/source/g/gevent/gevent-1.0.tar.gz" 
+pip install gevent
 echo "Installing supervisor"
 pip install supervisor
 echo "Installing GECOS Control Center UI"
 # Add --no-deps to speed up gecos-cc reinstallations and dependencies are already satisfied
-pip install --upgrade --force-reinstall "https://github.com/gecos-team/gecoscc-ui/archive/$GECOSCC_VERSION.tar.gz"
+pip install --upgrade "https://github.com/gecos-team/gecoscc-ui/archive/$GECOSCC_VERSION.tar.gz"
```

Comment:
"--force-reinstall" is needed if you are testing development branch of gecoscc. Perhaps is a bug of previous Python 2.6 but without this parameter, changes in gecoscc git repository does not overwrite the code installed in virtualenv (if you are using the same gecoscc version number).